website/docs/namespaced-mode.md

```diff
@@ -0,0 +1,116 @@
+---
+id: namespaced-mode
+title: Namespaced Mode
+sidebar_label: Namespaced Mode
+original_id: namespaced-mode
+---
+
+---
+
+### What is Namespaced Mode?
+
+Namespaced mode is the other end of the orchestration spectrum & is an antithesis of the admin-mode of operation. Here, the ChaosOperator as well as the chaos resources (chaosexperiment, chaosengine, chaosresult CRs and the experiment pods) are run in the same namespace where the application under test (AUT) resides. This mode serves the use-cases where the developers/users don't typically have much autonomy over cluster usage and operate strictly within a specific allotted namespace. However, in this case, too, the cluster-admin or an equivalent user with the right permissions are required to install them CRDs upfront.
+
+Note that this mode of operation restricts the scope of chaos experiments to those at the pod-level, as the node/infra level experiments need cluster-wide access.
+
+### How to use Namespaced Mode?
+
+In order to use Namespaced Mode, you just have to install the namespace-scoped litmus operator manifest in the desired (app) namespace. This installs the ChaosOperator, configured to watch for ChaosEngine resources in the same namespace while also setting up the RBAC that can be used for the execution of the standard (generic suite) of chaos experiments.
+
+#### Setup the Namespace-Scoped Chaos Operator
+
+- Install the chaos CRDs separately via an admin account
+
+  ```
```

Comment:
```suggestion
  ```bash
```

Reply:
You have provided just a basic fenced code block, I would suggest specifying bash to include syntax highlighting as well